ArrayMethods_Lab.js

```diff
@@ -0,0 +1,282 @@
+//# Exercises
+
+//1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+
+// ```
+// Input: [1,2,3,4,5]
+// Output: [11,12,13,14,15]
+// ```
+
+const upTen = arr => {
+    return arr.map(el => {
+        return el += 10
+  })
+}
+console.log(upTen([1,2,3,4,5]))
+
+//2. Given an array, return a new array where every non-string element is converted to an empty string.
+
+// ```
+// Input: ["a", 123, "b", "c", {name: "cat"}]
+// Output: ["a", "", "b", "c", ""]
+// ```
+
+const emptyString = arr => {
+    return arr.map(el => {
+    if (typeof(el) !== "string"){
+        return el = ""
+    } else {
+        return el
+    }   
+  })
+}
+console.log(emptyString(["a", 123, "b", "c", {name: "cat"}]))
+
+// 3. Write a function that returns only the even elements in an array of numbers.
+
+// ```
+// Input: [1,2,3,4,5]
+// Output: [2,4]
+// ```
+
+const evenArray = arr => {
+    let newArr = []
+    arr.forEach (el => {
+        if (el%2 === 0){
+        newArr.push(el)
+    }
+  })
+return newArr
+}
+console.log(evenArray([1,2,3,4,5])) 
+
+//const evenArray2  = arr => arr.map(el => el%2 === 0) // ** Don't use forEach to solve
+//console.log(evenArray2([1,2,3,4,5]))
+//evenArray2([1,2,3,4,5])
+
+// 4. Write a function that checks if all elements in the array are the same.
+
+// ```
+// Input: [1,1,1,1,1]
+// Output: true
+
+// Input: [1,1,1,1,2]
+// Output: false
+// ```
+
+const sameArr = arr => {
+    return arr.every((el , i) => {
```

Comment:
This can be done with only one every that checks if the el is the same as the arr[0]. Good job still!